test/stream.js

```diff
@@ -268,5 +268,17 @@ describe('gulp-ng-config', function () {
           done();
         }));
     });
+    it ('should generate pretty-looked content', function (done) {
+      //TODO i didn't know how to check line-breaks correctly
```

Comment:
What you have below looks correct, it's just that your `output_12.js` file uses 4 space tabs, instead of 2 spaced tabs. Just reduce them to 2 :)